site/cypress/e2e/Cypress-Feature/test_forum.spec.js

```diff
@@ -45,6 +46,29 @@ const removeThread = (title) => {
     cy.get('.thread-left-cont > .thread-list-item').contains(title).should('not.exist');
 };
 
+const editThread = (title) => {
+    cy.get('[data-testid="thread-list-item"]').contains(title).click();
+    cy.get('[data-testid="create-post-head"]').should('contain', title);
+    cy.get('[data-testid="edit-post-button"]').first().click();
+    cy.get('[data-testid="upload1"]').find('[for="input-file1"]').selectFile('cypress/fixtures/sea_animals.png');
+    cy.get('[data-testid="file-upload-table-1"]').find('.file-label').should('contain', 'sea_animals.png');
+    cy.get('[data-testid="file-upload-table-1"]').click();
+    cy.get('[data-testid="file-upload-table-1"]').should('not.contain');
+    cy.get('[data-testid="cancel-forum-btn"]').click();
+};
+const uploadAttachmentAndDelete = (title) => {
+    cy.get('[data-testid="thread-list-item"]').contains(title).click();
+    cy.get('[data-testid="create-post-head"]').should('contain', title);
+    cy.get('[data-testid="edit-post-button"]').first().click();
+    cy.get('#upload1').find('[for="input-file1"]').selectFile('cypress/fixtures/sea_animals.png');
+    cy.get('[data-testid="file-upload-table-1"]').find('.file-label').should('contain', 'sea_animals.png');
```

Comment:
Can you create a test-id for the other objects tested in this file, since it is a small test.

Reply:
done